hack/velero/deploy-velero.sh

```diff
@@ -58,6 +58,8 @@ if [[ ! $(_kubectl get deployments -n velero | grep velero) ]]; then
     --bucket velero \
     --secret-file ${velero_resources_dir}/credentials-velero \
     --use-volume-snapshots=true \
+    --velero-pod-mem-request 512Mi \
+    --velero-pod-mem-limit 1Gi \
```

Comment:
This is a high amount, are the OOMs maybe concerning to us/velero?
Maybe related to https://issues.redhat.com/browse/OCPBUGS-35436?